content/docs/scaffolder/writing-templates/index.md

```diff
@@ -1890,7 +1890,7 @@ This error is referring to actions that interact GitHub. It means that the Roadi
 
 Full error message: `Pull request creation failed; caused by HttpError: You have exceeded a secondary rate limit. Please wait a few minutes before you try again. If you reach out to GitHub Support for help, please include the request ID`
 
-This happens when you try to make a PR to Github with too many files in the PR, triggering a Secondary rate limit in Github's api. 
+This happens when you try to make a PR to Github with too many files in the PR, triggering a Secondary rate limit in Github's api. This is currently an issue with the open source action [explained in detail here](https://github.com/backstage/backstage/issues/17188).
```

Comment:
Could you mention this to engineers to subscribe to this issue so we know when to remove this doc line